BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.m

```diff
@@ -413,6 +435,33 @@ - (void)sendLogoutNotification
 #pragma mark - NSURLConnectionDataDelegate
 - (void)connection:(NSURLConnection *)connection didReceiveResponse:(NSURLResponse *)response
 {
+    [self processResponse:response];
```

Comment:
no changes to those delegate methods of NSURLConnectionDataDelegate, just take its content into a method to be reused for handling completion of nsurlsessiontask